### DISCUSSION TITLE
cursor up/down in file picker without arrow keys

### DISCUSSION THREAD

**Discussion by soomtong** [helix-editor/helix#2923] [Q&A] [answered]
Hi. a HHKB user here. many of us try to not use arrow keys as u know.
I searched disscussions and issues for file picker action. but I can't get a ticket.
is there a way to navigate a list of files in file picker?
I know to use fuzzy search by typing letters.

![image](https://user-images.githubusercontent.com/906974/176675175-62e95509-a54f-465f-a91f-217c340dbca9.png)

in my case I usually use `ctrl+j/k` to doing this action. can I get a plan or some way do this?

**Comment by the-mikedavis** [comment, marked as answer]
`C-n` and `C-p` work on pickers as well as the autocompletion menu.

**Comment by soomtong** [comment]
Thank you for answer on my useful but stupid question!

**Comment by nestor-custodio** [comment]
See also [the keymap documentation](https://docs.helix-editor.com/keymap.html#picker), which has a more complete list of available keybindings in the file picker.

**Comment by bonsairobo** [comment]
Tab and Shift-Tab also work.
